src/k8s/pkg/k8sd/database/schema.go

```diff
@@ -18,9 +18,11 @@ var (
 	SchemaExtensions = []schema.Update{
 		schemaApplyMigration("kubernetes-auth-tokens", "000-create.sql"),
 		schemaApplyMigration("cluster-configs", "000-create.sql"),
+		schemaApplyMigration("worker-nodes", "000-create.sql"),
 		schemaApplyMigration("worker-tokens", "000-create.sql"),
 		schemaApplyMigration("feature-status", "000-feature-status.sql"),
 		schemaApplyMigration("worker-tokens", "001-add-expiry.sql"),
+		schemaApplyMigration("worker-nodes", "002-delete.sql"),
```

Comment:
I totally get what we're going for here. but since we have directories for different domains (tables), it might be confusing to share a global ordering between these domains (tables). I guess we'll soon have something like this in the `src/k8s/pkg/k8sd/database/sql/migrations` which I think is confusing and probably not necessary (?):
```
.
├── cluster-configs
│   ├── 000-create.sql
│   └── 004-something-else.sql
├── feature-status
│   ├── 000-feature-status.sql
│   └── 003-something.sql
├── kubernetes-auth-tokens
│   ├── 000-create.sql
│   └── 005-another-thing.sql
├── worker-nodes
│   ├── 000-create.sql
│   └── 002-delete.sql
└── worker-tokens
    ├── 000-create.sql
    └── 001-add-expiry.sql
```
the only thing that we need to do right now is to make sure we don't delete and re-order the migrations again and keep the orderings in the scope of the domain (table). this is more like a personal preference from me tho, nothing super important.

Reply:
This numbering is giving a false idea of order... You are right the sequence is confusing as well, I'll drop the number to 001 for the new migration.